src/encoded/static/components/matrix_human_donor.js

```diff
@@ -165,11 +166,14 @@ const convertToMatrixData = (dataTableData, baseUrl, searchBaseUrl, graph) => {
             ),
             itemKey: 'matrix-header-key-titles',
         }].concat(dataTableData.x.header.map((xData) => {
-            const assayTitleText = `assay_title=${xData}`;
-            const assayTitle = baseUrl.includes(assayTitleText) ? `&${assayTitleText}` : '';
+            const baseUrlQuery = new QueryString(baseUrl);
+            baseUrlQuery.deleteKeyValue('assay_title');
+            baseUrlQuery.addKeyValue('assay_title', xData);
```

Comment:
There’s also a `replaceKeyValue`, though all it does is the same two steps you have here.

Reply:
Good point.  I used deleteKeyValue because there could be multiple instances of assay_title query string and I need to delete them all. So, replaceKeyValue will lead to a bug if used.